css/style.css

```diff
@@ -0,0 +1,99 @@
+
+body {
+  background-color: grey;
+  display: grid;
+  font-family: sans-serif;
+	font-size: 24px;
+	margin: 0;
+  min-height: 100vh;
+  grid-template-columns: 10% 80% 10%;
+  grid-template-rows: 20% 50% 30%;
+  grid-template-rows: 
+  "null title null"
+  "null main null"
+  "null footer null";
```

Comment:
You're trying to set up `grid-template-areas` but using `grid-template-rows` on line 11